### PR TITLE
add more team data, refactor team card

### PR DESCRIPTION
**Pull request by sjpaige**
[icon bleeding bug](https://airtable.com/appfJZShN8K4tcWHU/tblvBIYoi6TLmdRAv/viwdz1LjpFf9DmApF/rec7Sy9Su5O5q6MkT?blocks=hide)

[add others bug](https://airtable.com/appfJZShN8K4tcWHU/tblvBIYoi6TLmdRAv/viwdz1LjpFf9DmApF/recIcJlqChHWlvZmP?blocks=hide)

- [x] refactor team card to look better across browsers
- [x] add more team data
- [x] collect team photos
- [x] host photos on s3
- [x] handle requests for photos
- [ ] get more data from team members make sure everyone is included

### CODE REVIEW

**Comment by sjpaige** [comment]
> Ooh! There's a "bug" for this, want to pick it up? https://airtable.com/appfJZShN8K4tcWHU/tblvBIYoi6TLmdRAv/viwdz1LjpFf9DmApF/recIcJlqChHWlvZmP?blocks=hide

@stuttskl Yeah there were two relevant bug reports on there this should address both of them. We don't need to address the hosting situation in this PR but I wanted to make sure I was listing everything.

**Comment by sjpaige** [comment]
@dylankershaw Do you have any thoughts on how I should go about adding this in for the team cards?

**Comment by sjpaige** [comment]
@dylankershaw I may have put the lazy load too high in the tree, and the fallback is a little simple right now, but please take a look at my solution and tell me what you think.

**Comment by dylankershaw** [comment]
> @dylankershaw I may have put the lazy load too high in the tree, and the fallback is a little simple right now, but please take a look at my solution and tell me what you think.
> 
> also I have the permissions completely unlock right now but once its deployed to prod we can re-enable the bucket policy.

It looks like the Suspense API is experimental and might not be reliable for production use: https://reactjs.org/docs/concurrent-mode-suspense.html

Also apparently HTML has built-in lazy loading for all browsers except Safari and IE: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading

There's also a bunch of other libraries we can use: https://blog.logrocket.com/the-top-choices-for-react-lazy-loading-libraries-in-2021/

Maybe we should hold off on lazy loading for launch and then revisit this in the next sprint? This would give us more time to pick a suitable option.

**Comment by sjpaige** [comment]
> > @dylankershaw I may have put the lazy load too high in the tree, and the fallback is a little simple right now, but please take a look at my solution and tell me what you think.
> 
> > 
> 
> > also I have the permissions completely unlock right now but once its deployed to prod we can re-enable the bucket policy.
> 
> 
> 
> It looks like the Suspense API is experimental and might not be reliable for production use: https://reactjs.org/docs/concurrent-mode-suspense.html
> 
> 
> 
> Also apparently HTML has built-in lazy loading for all browsers except Safari and IE: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading
> 
> 
> 
> There's also a bunch of other libraries we can use: https://blog.logrocket.com/the-top-choices-for-react-lazy-loading-libraries-in-2021/
> 
> 
> 
> Maybe we should hold off on lazy loading for launch and then revisit this in the next sprint? This would give us more time to pick a suitable option.


@dylankershaw Okay sounds good. I will remove the lazy load for now.

File: src/Components/TeamCard.jsx [hunk 1]
```diff
@@ -2,49 +2,49 @@ import {
...
-  const {firstName, lastName, role, photo, linkedIn} = props.member;
 
   return (
     <Card className={classes.card}>
       <CardMedia>
-        <Avatar className={classes.avatar} variant="square" src={photo} />
+        <Avatar
+          className={classes.avatar}
+          variant="square"
+          src={`https://cvp-team-photos.s3.us-west-2.amazonaws.com/${firstName+lastName}.jpg`}
```
Comment (stuttskl) [inline src/Components/TeamCard.jsx:37]: Nice! We should add documentation somewhere that specifies how new team member photos need to be named when being added to the s3 bucket.

File: src/data/teamData.ts [hunk 2]
```diff
@@ -1,11 +1,131 @@
-import Seamus_Brugh from "../Assets/team/Seamus Brugh.png";
-const team = [
+
```
Comment (stuttskl) [inline src/data/teamData.ts:1]: extra space

File: src/data/teamData.ts [hunk 3]
```diff
@@ -1,11 +1,131 @@
...
+type teamMember = {
```
Comment (stuttskl) [inline src/data/teamData.ts:2]: I think custom types and interfaces are customarily capitalized. So instead it would be `TeamMember`

**Review by stuttskl — CHANGES_REQUESTED** [review]

File: src/data/teamData.ts (hunk 3 above)
Comment (sjpaige) [reply src/data/teamData.ts:2]: Ah good catch.

File: src/Components/TeamCard.jsx (hunk 1 above)
Comment (sjpaige) [reply src/Components/TeamCard.jsx:37]: I guess we could have it in the AWS document that dylan created, unless we want to create another.

Comment (sjpaige) [reply src/Components/TeamCard.jsx:37]: Something like [this](https://docs.google.com/document/d/19M9ErJop-09XDQHRmEWhI6EZX3k-2iSrlvUiSgK5OnY/edit)

Comment (stuttskl) [reply src/Components/TeamCard.jsx:37]: Yeah, that's perfect

**Review by stuttskl — APPROVED** [review]

**Pull request merged**